monai/transforms/io/array.py

```diff
@@ -204,6 +204,8 @@ def __call__(self, filename: Union[Sequence[PathLike], PathLike], reader: Option
                         break
 
         if img is None or reader is None:
+            if isinstance(filename, tuple) and len(filename) == 0:
```

Comment:
May I know why the length equals to `0`?

Thanks.

Reply:
Apologies, should be 1!